app/jobs/update_developer_response_rate_job.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateDeveloperResponseRateJob < ApplicationJob
+  queue_as :default
+
+  GRACE_PERIOD = Rails.application.config.developer_response_grace_period
+
+  def perform(developer)
+    conversations = developer.conversations.where("created_at < ?", GRACE_PERIOD.ago)
+    replied_rate = Stats::Conversation.new(conversations).replied_rate
+    developer.update_column(:response_rate, (replied_rate * 100).round.floor(-1))
```

Comment:
I don't think this logic is correct. Isn't it only querying for conversations started within the last 12 hours?

Instead we need something that _ignores_ all conversations created in the last 12 hours and calculates the response rate of the others.

Reply:
`developer.conversations.where("created_at < ?", GRACE_PERIOD.ago)`
Fetch conversations where `created_at` is less than (i.e earlier than) the given time value.

I've added tests to confirm the logic.

Reply:
Ah! You're right. I read that backwards. Thanks for clarifying - and with tests to prove it.